Supervisor comments (#54)
chhoumann/p8: report/src/presence/our_choice.tex

\section{Choice of technology}
In this Section, we will compare the technologies presented in Sections \ref{sec:presence_env} to \ref{sec:bluetooth}, and decide which technology best satisfies the criterias listed in Table \ref{lst:presence_eval_criteria}. 

Section \ref{sec:presence_env}, presents $\text{CO}_{2}$ sensoring as an occupation detection technology. The technology can accurately detect the presence in a room using by measuring $\text{CO}_{2}$ levels.

> The technology sufffer from unpredictability due to it being dependent on the environmental context of the system.

Reviewer: ```suggestion
The technology suffers from unpredictability, due to its dependency on the environmental context of the system.
```
